includes/assets.php

```diff
@@ -41,7 +41,7 @@ function fastwc_admin_enqueue_scripts() {
 		$select2_version
 	);
 
-	$fast_js_version = '1.0.0';
+	$fast_js_version = '1.0.8';
```

Comment:
eight patch versions? or does this just match the whole plugin now?

Reply:
My intent here is to just match the plugin version number. It's much easier than trying to keep up with the JS number. the main reason here is that WordPress appends the version number when the JS is loaded (like `script.js?v=1.0.8` so that if/when the JS is updated, the new version gets loaded instead of a cached version.